website/docs/guides/advanced-custom-extractor.md

```diff
@@ -0,0 +1,36 @@
+# Advanced: Custom Extractor
```

Comment:
```suggestion
# Custom Extractor
```

I'd suggest removing this prefix since such a heading format is something unusual for this site

\+ rename file

Reply:
That's exactly how i wanted to write this. There will be more in the future. If you want you can create a separate group in the sidebar for that. But this is totally fine to have "advanced" https://nextjs.org/docs/advanced-features/compiler

Reply:
Let's create a separate "Advanced" category. This could be a sub-category inside the "Guides" category.